app/controllers/change_answers_controller.rb

```diff
@@ -11,12 +11,15 @@ def update
       session_data.merge!(@form.attributes_for_export_to_session)
       if FeatureFlags.enabled?(:ee_banner, session_data)
         next_step = step_with_inconsistent_data
-        if Steps::Helper.cannot_use_service?(session_data, step)
+        if Steps::Logic.landlord_is_not_the_only_joint_owner?(session_data)
+          redirect_to cannot_use_service_path assessment_code:, step:
```

Comment:
@willc-work the logic here seems to be getting quite complex.

I am also not sure it is correct/will work the way you intend?

The cannot_use_service_path assessment_code:, step: route is currently only for when the client is exited because they or their partner have an additional shared ownership property.

When the applicant is exited because they have a landlord who is not the sole owner this uses the cannot_use_service_path assessment_code: path (ie it doesn't use the cannot_use_service controller - the work to do that is in EL-2159)

So I think this pr will currently take the user to the wrong exit page?

Reply:
I think we can reuse the same page to exit users, although that may impact metrics, is that your concern? The ticket just says :
`Selecting ‘No’ to 'Is the landlord the only other joint owner?' does not direct you to an exit page.`
There is no specification of what the exit page should be so I have assumed the existing exit page is to be used. I will check this with Tob

Reply:
@willc-work No there are multiple exit pages. Do you want to discuss.